core/include/kernel/transfer_list.h

```diff
@@ -6,7 +6,7 @@
 #ifndef __KERNEL_TRANSFER_LIST_H
 #define __KERNEL_TRANSFER_LIST_H
 
-#define TRANSFER_LIST_SIGNATURE		U(0x006ed0ff)
+#define TRANSFER_LIST_SIGNATURE		U(0x4a0fb10b)
```

Comment:
I think the backward compatibility is not yet considered at this moment since current v0.9 is still not a stable version (The rev doesn't increase with changes).

Below is the related PR for the spec:
https://github.com/FirmwareHandoff/firmware_handoff/pull/24

Reply:
Ouch! Good thing that this is by default off in all platforms since we usually don't break ABIs. I guess that means that it should remain off until we're using a stable version. It might be worth mentioning that in `mk/config.mk`.

Reply:
Comment is added.